crates/bitcoin-da/src/service.rs

```diff
@@ -744,11 +745,10 @@ mod tests {
     async fn check_signature() {
         let da_service = get_service().await;
         let secp = bitcoin::secp256k1::Secp256k1::new();
-        let da_pubkey =
-            Keypair::from_secret_key(&secp, &da_service.sequencer_da_private_key.unwrap())
-                .public_key()
-                .serialize()
-                .to_vec();
+        let da_pubkey = Keypair::from_secret_key(&secp, &da_service.da_private_key.unwrap())
```

Comment:
If da_service.da_private_key is required then it shouldn't be Option anymore

Reply:
that's a test + private key should be option, full node does not need to supply that